api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -113,8 +113,29 @@ type ClusterGroupUpgradeSpec struct {
 	//   - label1Name
 	// All the clusters matching the labels specified in clusterSelector will be included
 	// in the update plan.
+	// Deprecated: Use ClusterLabelSelectors instead
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Cluster Selector",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	ClusterSelector []string `json:"clusterSelector,omitempty"`
+	// This field holds a list of expressions or labels that will be used to determine what clusters to include in the operation.
+	// The expected format is as follows:
+	// clusterLabelSelectors:
+	//   - matchExpressions:
+	//       - key: label1
+	//         operator: In
+	//         values:
+	//           - value1a
+	//           - value1b
+	//   - matchLabels:
+	//       label2: value2
+	//   - matchExpressions:
+	//       - key: label3
+	//         operator: In
+	//         values:
+	//           - value3
+	//     matchLabels:
```

Comment:
Is this "matchLabels" missing a leading "-", eg:
```suggestion
	//    - matchLabels:
```

Reply:
No, this third one is just an example of a label selector that includes both expression and label elements, compared to the first two which are respectively just an expression and just a label